matlab/test/arrow/tabular/tRecordBatch.m

```diff
@@ -119,7 +119,21 @@ function UnsupportedColumnIndexType(tc)
             TOriginal = table(1, 2, 3);
             arrowRecordBatch = arrow.recordbatch(TOriginal);
             fcn = @() arrowRecordBatch.column(datetime(2022, 1, 3));
-            tc.verifyError(fcn, "arrow:tabular:recordbatch:UnsupportedColumnIndexType");
+            tc.verifyError(fcn, "arrow:badsubscript:NonNumeric");
+        end
+
+        function ErrorIfIndexIsNonScalar(tc)
+            TOriginal = table(1, 2, 3);
+            arrowRecordBatch = arrow.recordbatch(TOriginal);
+            fcn = @() arrowRecordBatch.column([1 2]);
+            tc.verifyError(fcn, "MATLAB:expectedScalar");
+        end
+
+        function ErrorIfNonPositiveIndex(tc)
```

Comment:
To be consistent with the naming of `ErrorIfIndexIsNonScalar`, perhaps we should rename this to `ErrorIfIndexIsNonPositive`?

Reply:
Good point.